Remember-iOS/Remember-iOS/Network/APIService/MainService.swift

```diff
@@ -6,3 +6,52 @@
 //
 
 import Foundation
+
+import Moya
+
+enum MainService {
+    case getCategoryData
+    case getMainData
```

Comment:
enum에서 case를 네이밍을 할 때 동사로 시작하는 이름보다는 명사로 떨어지는 이름이 좋아요!
```suggestion
    case category
    case main
```
`getXXXXDate`는 나중에 해당 데이터를 받는 서버 함수 네이밍으로 더 적합할 것 같아요!

Reply:
오..그렇군여..당장 고치러 가겠읍니다,,
조언 감사해요오🔥